package.json

```diff
@@ -7,7 +7,7 @@
     "start": "ts-node-dev src/app.ts",
     "latest": "knex migrate:latest --knexfile knexfile.ts migrate:latest",
     "seed": "knex --knexfile knexfile.ts seed:run",
-    "clean": "rm -f src/database/database.sqlite",
+    "clean": "del -f src\\database\\database.sqlite",
```

Comment:
This should be a secondary script option for cleaning. The previous code should be kept as it is work as intendend on linux environments.